CMakeLists.txt

```
@@ -87,6 +87,7 @@ if(WIN32)
   # On Windows, CMAKE_HOST_SYSTEM_PROCESSOR is calculated through `PROCESSOR_ARCHITECTURE`,
   # which only has the value of `x86` or `AMD64`. We cannot infer whether it's a Intel CPU
   # or not. However, the environment variable `PROCESSOR_IDENTIFIER` could be used.
+  set(CPU_AARCH64 OFF)
```

Let's give CPU_AARCH64 and CPU_INTEL a default value here and then only set it to true in conditional.
```suggestion
  set(CPU_AARCH64 OFF)
   set(CPU_INTEL OFF)

```